### DISCUSSION TITLE
Suggestion for pricing

### DISCUSSION THREAD

**Discussion by qqwxp** [reworkd/AgentGPT#745] [Ideas]
Please introduce a mid-tier subscription plan or provide educational discounts for the subscription service to make it more affordable and accessible. Thank you.

**Comment by jasangill1** [comment]
Hello @qqwxp What price range did you have in mind, along with what would be a fair feature / limits for that price would love to hear from you the user !!

**Reply by qqwxp** [reply]
Hi @jasangill1 ,

Thank you for your response and consideration of my suggestion. I believe a $20/month tier would be more appealing to a broader audience, as this price point aligns with the majority of similar products in the market. For this price, I'd suggest offering a mid-tier set of features and limits, perhaps including some advanced functionalities but excluding certain premium or specialized features. This could cater to the needs of most users and potentially attract a larger subscriber base.

Once again, I appreciate your openness to feedback and look forward to seeing the continued growth and improvements of your product.

**Reply by jasangill1** [reply]
Great idea @qqwxp will take this up with the team!! Please feel free to message me with any other suggestions or ideas you may have!!!
